README.md

```diff
@@ -112,7 +112,7 @@ _setting.yaml_
 PSYDUCK: yaml
 ```
 
-Примеры:
+Examples:
```

Comment:
[MD003 - Heading style](https://github.com/DavidAnson/markdownlint/blob/v0.13.0/doc/Rules.md#md003---heading-style)
[MD026 - Trailing punctuation in heading](https://github.com/DavidAnson/markdownlint/blob/v0.13.0/doc/Rules.md#md026)
```suggestion
## Examples

```